### ISSUE TITLE
remonté les anomalies de la chaudiere

### ISSUE THREAD

**Issue by stawen** [stawen/okovision#67]
recuperer la liste des anomalie et les afficher dans okovision

**Comment by bertrandgorge** [comment]
Il y a une API pour ça ?

**Comment by stawen** [comment]
oui, c'est le meme mecanisme que pour la partie temps réel d'okovision. la chaudiere fourni une api REST sur laquel l'appli web interne de la chaudiere s'appuie, il faut juste trouver la bonne requette car il n'y a aucune doc.

**Comment by bertrandgorge** [comment]
Si tu as des détails sur ça, je peux essayer de regarder...!
